_posts/2024-01-11-gsoc2023_marinegor.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: post
+title: GSoC 2023 - Implementation of parallel analysis in MDAnalysis
+---
+
+As you might know from my previous posts, during the summer of 2023 I've been working on MDAnalysis's project during Google Summer of Code. Here I'll summarize what I've done, how others can use it, and what changes will follow that in the MDAnalysis codebase in the near future.
```

Comment:
could you add link to the blog in "previous posts"?
https://marinegor.github.io/year-archive/

Reply:
Done.